lib/route_types.txt

```diff
@@ -57,6 +57,7 @@ ZEBRA_ROUTE_OSPF6,      ospf6,     ospf6d, 'O', 0, 1, 1,     "OSPFv3",        os
 ZEBRA_ROUTE_ISIS,       isis,      isisd,  'I', 1, 1, 1,     "IS-IS",         isisd
 ZEBRA_ROUTE_BGP,        bgp,       bgpd,   'B', 1, 1, 1,     "BGP",           bgpd
 ZEBRA_ROUTE_PIM,	pim,	   pimd,   'P', 0, 0, 0,     "PIM",           pimd
+ZEBRA_ROUTE_PIM6,	pim6,	   pim6d,  'P', 0, 0, 0,     "PIM6",          pim6d
```

Comment:
Why do we need to introduce another protocol, why not just leave it ZEBRA_ROUTE_PIM?